{
  "uid" : "c467ffae46fe97f4",
  "name" : "TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation",
  "fullName" : "tests.test_sign_in.TestFailedLogin#test_03_02_01_to_03_02_10_login",
  "historyId" : "7f91f4a63115fb89e466c8c9b85c1503",
  "time" : {
    "start" : 1729999105672,
    "stop" : 1729999405691,
    "duration" : 300019
  },
  "description" : "Check Success and Failed Login, no error validation",
  "descriptionHtml" : "<p>Check Success and Failed Login, no error validation</p>\n",
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.592\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x56324ba050aa <unknown>\n#1 0x56324b51c1a0 <unknown>\n#2 0x56324b504840 <unknown>\n#3 0x56324b50452d <unknown>\n#4 0x56324b5025bc <unknown>\n#5 0x56324b502d6f <unknown>\n#6 0x56324b5121d7 <unknown>\n#7 0x56324b5281e4 <unknown>\n#8 0x56324b52d51b <unknown>\n#9 0x56324b50347e <unknown>\n#10 0x56324b527fee <unknown>\n#11 0x56324b5abacb <unknown>\n#12 0x56324b58cb23 <unknown>\n#13 0x56324b55b990 <unknown>\n#14 0x56324b55c96e <unknown>\n#15 0x56324b9d116b <unknown>\n#16 0x56324b9d4f68 <unknown>\n#17 0x56324b9be64c <unknown>\n#18 0x56324b9d5ae7 <unknown>\n#19 0x56324b9a34af <unknown>\n#20 0x56324b9f34f8 <unknown>\n#21 0x56324b9f36c0 <unknown>\n#22 0x56324ba03f26 <unknown>\n#23 0x7f56ade94ac3 <unknown>",
  "statusTrace" : "self = <test_sign_in.TestFailedLogin object at 0x7fc6ec70e1d0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3478baa83c8f83cca63f497e7a416ed2\")>\nemail = 'teston120@mailinator.com', password = 'test#12p@ssword'\n\n    @allure.title('TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation')\n    @pytest.mark.parametrize('email, password', LOGIN)\n    def test_03_02_01_to_03_02_10_login(self, driver, email, password):\n        \"\"\"Check Success and Failed Login, no error validation\"\"\"\n        sign_in_page = SignInPage(driver, SIGN_IN_URL)\n>       sign_in_page.open()\n\ntests/test_sign_in.py:255: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/base_page.py:19: in open\n    self.driver.get(self.url)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fc6edd371d0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"timeout\",\"message\":\"timeout: Timed out receiving message from renderer: 2...\\\\n#21 0x56324b9f36c0 \\\\u003Cunknown>\\\\n#22 0x56324ba03f26 \\\\u003Cunknown>\\\\n#23 0x7f56ade94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.592\nE         (Session info: chrome=130.0.6723.58)\nE       Stacktrace:\nE       #0 0x56324ba050aa <unknown>\nE       #1 0x56324b51c1a0 <unknown>\nE       #2 0x56324b504840 <unknown>\nE       #3 0x56324b50452d <unknown>\nE       #4 0x56324b5025bc <unknown>\nE       #5 0x56324b502d6f <unknown>\nE       #6 0x56324b5121d7 <unknown>\nE       #7 0x56324b5281e4 <unknown>\nE       #8 0x56324b52d51b <unknown>\nE       #9 0x56324b50347e <unknown>\nE       #10 0x56324b527fee <unknown>\nE       #11 0x56324b5abacb <unknown>\nE       #12 0x56324b58cb23 <unknown>\nE       #13 0x56324b55b990 <unknown>\nE       #14 0x56324b55c96e <unknown>\nE       #15 0x56324b9d116b <unknown>\nE       #16 0x56324b9d4f68 <unknown>\nE       #17 0x56324b9be64c <unknown>\nE       #18 0x56324b9d5ae7 <unknown>\nE       #19 0x56324b9a34af <unknown>\nE       #20 0x56324b9f34f8 <unknown>\nE       #21 0x56324b9f36c0 <unknown>\nE       #22 0x56324ba03f26 <unknown>\nE       #23 0x7f56ade94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1729999104420,
      "stop" : 1729999105672,
      "duration" : 1252
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder",
    "time" : {
      "start" : 1729993092064,
      "stop" : 1729993092065,
      "duration" : 1
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "_session_faker",
    "time" : {
      "start" : 1729993092000,
      "stop" : 1729993092064,
      "duration" : 64
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "testStage" : {
    "description" : "Check Success and Failed Login, no error validation",
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.592\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x56324ba050aa <unknown>\n#1 0x56324b51c1a0 <unknown>\n#2 0x56324b504840 <unknown>\n#3 0x56324b50452d <unknown>\n#4 0x56324b5025bc <unknown>\n#5 0x56324b502d6f <unknown>\n#6 0x56324b5121d7 <unknown>\n#7 0x56324b5281e4 <unknown>\n#8 0x56324b52d51b <unknown>\n#9 0x56324b50347e <unknown>\n#10 0x56324b527fee <unknown>\n#11 0x56324b5abacb <unknown>\n#12 0x56324b58cb23 <unknown>\n#13 0x56324b55b990 <unknown>\n#14 0x56324b55c96e <unknown>\n#15 0x56324b9d116b <unknown>\n#16 0x56324b9d4f68 <unknown>\n#17 0x56324b9be64c <unknown>\n#18 0x56324b9d5ae7 <unknown>\n#19 0x56324b9a34af <unknown>\n#20 0x56324b9f34f8 <unknown>\n#21 0x56324b9f36c0 <unknown>\n#22 0x56324ba03f26 <unknown>\n#23 0x7f56ade94ac3 <unknown>",
    "statusTrace" : "self = <test_sign_in.TestFailedLogin object at 0x7fc6ec70e1d0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"3478baa83c8f83cca63f497e7a416ed2\")>\nemail = 'teston120@mailinator.com', password = 'test#12p@ssword'\n\n    @allure.title('TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation')\n    @pytest.mark.parametrize('email, password', LOGIN)\n    def test_03_02_01_to_03_02_10_login(self, driver, email, password):\n        \"\"\"Check Success and Failed Login, no error validation\"\"\"\n        sign_in_page = SignInPage(driver, SIGN_IN_URL)\n>       sign_in_page.open()\n\ntests/test_sign_in.py:255: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/base_page.py:19: in open\n    self.driver.get(self.url)\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7fc6edd371d0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"timeout\",\"message\":\"timeout: Timed out receiving message from renderer: 2...\\\\n#21 0x56324b9f36c0 \\\\u003Cunknown>\\\\n#22 0x56324ba03f26 \\\\u003Cunknown>\\\\n#23 0x7f56ade94ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.592\nE         (Session info: chrome=130.0.6723.58)\nE       Stacktrace:\nE       #0 0x56324ba050aa <unknown>\nE       #1 0x56324b51c1a0 <unknown>\nE       #2 0x56324b504840 <unknown>\nE       #3 0x56324b50452d <unknown>\nE       #4 0x56324b5025bc <unknown>\nE       #5 0x56324b502d6f <unknown>\nE       #6 0x56324b5121d7 <unknown>\nE       #7 0x56324b5281e4 <unknown>\nE       #8 0x56324b52d51b <unknown>\nE       #9 0x56324b50347e <unknown>\nE       #10 0x56324b527fee <unknown>\nE       #11 0x56324b5abacb <unknown>\nE       #12 0x56324b58cb23 <unknown>\nE       #13 0x56324b55b990 <unknown>\nE       #14 0x56324b55c96e <unknown>\nE       #15 0x56324b9d116b <unknown>\nE       #16 0x56324b9d4f68 <unknown>\nE       #17 0x56324b9be64c <unknown>\nE       #18 0x56324b9d5ae7 <unknown>\nE       #19 0x56324b9a34af <unknown>\nE       #20 0x56324b9f34f8 <unknown>\nE       #21 0x56324b9f36c0 <unknown>\nE       #22 0x56324ba03f26 <unknown>\nE       #23 0x7f56ade94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py:229: TimeoutException",
    "steps" : [ {
      "name" : "Open a browser",
      "time" : {
        "start" : 1729999105672,
        "stop" : 1729999405690,
        "duration" : 300018
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 299.592\n  (Session info: chrome=130.0.6723.58)\nStacktrace:\n#0 0x56324ba050aa <unknown>\n#1 0x56324b51c1a0 <unknown>\n#2 0x56324b504840 <unknown>\n#3 0x56324b50452d <unknown>\n#4 0x56324b5025bc <unknown>\n#5 0x56324b502d6f <unknown>\n#6 0x56324b5121d7 <unknown>\n#7 0x56324b5281e4 <unknown>\n#8 0x56324b52d51b <unknown>\n#9 0x56324b50347e <unknown>\n#10 0x56324b527fee <unknown>\n#11 0x56324b5abacb <unknown>\n#12 0x56324b58cb23 <unknown>\n#13 0x56324b55b990 <unknown>\n#14 0x56324b55c96e <unknown>\n#15 0x56324b9d116b <unknown>\n#16 0x56324b9d4f68 <unknown>\n#17 0x56324b9be64c <unknown>\n#18 0x56324b9d5ae7 <unknown>\n#19 0x56324b9a34af <unknown>\n#20 0x56324b9f34f8 <unknown>\n#21 0x56324b9f36c0 <unknown>\n#22 0x56324ba03f26 <unknown>\n#23 0x7f56ade94ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 19, in open\n    self.driver.get(self.url)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.11.10/x64/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1729999705734,
      "stop" : 1729999705841,
      "duration" : 107
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "clear_allure_result_folder::0",
    "time" : {
      "start" : 1729999808072,
      "stop" : 1729999808072,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "feature",
    "value" : "Failed Login Functionality"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_sign_in"
  }, {
    "name" : "subSuite",
    "value" : "TestFailedLogin"
  }, {
    "name" : "host",
    "value" : "fv-az841-46"
  }, {
    "name" : "thread",
    "value" : "2812-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_sign_in"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "email",
    "value" : "'teston120@mailinator.com'"
  }, {
    "name" : "password",
    "value" : "'test#12p@ssword'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "c467ffae46fe97f4.json",
  "parameterValues" : [ "'teston120@mailinator.com'", "'test#12p@ssword'" ]
}